my_portfolio.html

```diff
@@ -1,13 +1,63 @@
 ﻿<!DOCTYPE html>
-
 <html lang="en">
+
 <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css" />
 </head>
+
 <body>
-    <h1>Hello world</h1>
+    <img src="icons/header-illustration-mobile.svg" id="header-img" />
+    <img src="icons/header-desktop.svg" id="header-img-desktop" />
+    <div id="nav-bar-ctn">
+        <div id="nav-bar">
+            <a>
+                <p>Portfolio</p>
+            </a>
+            <a>
+                <p>About</p>
+            </a>
+            <a>
+                <p>Contact</p>
+            </a>
+        </div>
+    </div>
+    <div id="toolbar">
+        <div id="menu-button">
+            <object data="icons/icon-menu.svg"></object>
+        </div>
+    </div>
```

Comment:
- [ ] Kindly consider wrapping this in the `header` tag because Typically the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info. Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.